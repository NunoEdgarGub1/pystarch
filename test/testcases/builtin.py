x01 = any(x == 0 for x in [1,2,3])
x02 = abs(-3)
x03 = enumerate(['a', 'b', 'c'])
