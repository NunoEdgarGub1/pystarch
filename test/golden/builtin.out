x01 Bool
x02 Num
x03 List(Tuple(Num,Str))
